platforms/netbsd/hax_entry_vcpu.c

```diff
@@ -58,6 +58,40 @@ struct cdevsw hax_vcpu_cdevsw = {
     .d_flag = D_OTHER | D_MPSAFE
 };
 
+#define load_user_data(dest, src, body_len, body_max, arg_t, body_t)          \
```

Comment:
Why not create a function? My ioctl will use it too.

Reply:
A function is better.. but I want to avoid divergence with other kernels.

Reply:
How does a function lead to divergence?

Reply:
Linux + Darwin use macro here.

Reply:
> Why not create a function?

This pair of function-like macros are designed for all interfaces with variable-length arguments. As there are types, struct members in the macro parameters, it is hard to define a real function to handle these cases. But I agree to bring the definition to a common place, such as `hax.h`, instead of duplicating the codes here.

> Linux + Darwin use macro here.

As for the minor difference between NetBSD/Darwin and Linux, should we merge them into one macro by platform switch?

Reply:
It's better to keep the macro in private files of kernels as there are OS-specific differences.